### PR TITLE
waku/faq

### PR DESCRIPTION
**Pull request by decanus**

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+permalink: /faq/
+title: FAQ
+---
+
+## Are there messaging protocols out there that you feel live up to your security and privacy Standards? Where does Signal fall in?
+
+<!-- @todo do we wanna keep this? its kinda status related -->
```
Comment (oskarth) [inline faq.md]: maybe rephrase it as "how does this compare with Signal"

**Comment by decanus** [comment]
@oskarth do you have any other things we could add here?

File: faq.md (hunk 1 above)
Comment (decanus) [reply faq.md]: @oskarth look at  https://github.com/vacp2p/vacp2p.github.io/pull/13/commits/5fb09da52489a0a689404f50db278516244a9c52, does this make more sense?

File: faq.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Waku is more scalable than Whisper, but it still has a lot of issues. We aim to address these in an incremental manner 
+until it works as a long term solution and stands the test of time.
+
+## What are the key privacy tradeoffs between Waku and Whisper?
+
+First off, the privacy guarantees provided by Whisper are mostly theoretical at this stage. They've not received a lot 
+of scrutiny from academic researchers and so on.
+
+There are a few trade-offs that allow Waku to be more scalable and suitable for resource restricted devices than Whisper. 
+It is worth noting that all of these are optional, and a Waku client can choose to get all the privacy benefits by turning these options off:
```
Comment (oskarth) [inline faq.md]: What about linking to the part of the spec that talks about this?

**Comment by oskarth** [comment]
Seems fine for now, is the Draft mode deliberate?
